adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -101,6 +106,11 @@ workflow AdapterOptimus {
   File annotations_gtf  # gtf containing annotations for gene tagging
   File ref_genome_fasta  # genome fasta file
   String fastq_suffix = ".gz"  # add this suffix to fastq files for picard
+  
+  # Note: This "Empty" is a workaround in WDL-draft to simulate a "None" type
+  # once the official "None" type is introduced (probably in WDL 2.0)
+  # we should consider replace this workaround
+  Array[File]? Empty
```

Comment:
Maybe call it `None`? Will make migrating to the real `None` even easier, when it arrives.
```suggestion
  Array[File]? None
```

Reply:
Good point, will do!